main/src/ca/uwaterloo/flix/language/phase/Weeder.scala

```diff
@@ -1134,9 +1134,20 @@ object Weeder {
         case err: WeederError => WeededAst.Expr.Error(err)
       }
 
-    case ParsedAst.Expression.LetRecDef(sp1, ident, fparams, tpeAndEff, exp1, exp2, sp2) =>
+    case ParsedAst.Expression.LetRecDef(sp1, ann, ident, fparams, tpeAndEff, exp1, exp2, sp2) =>
       val mod = Ast.Modifiers.Empty
       val loc = mkSL(sp1, sp2)
+      val annVal = flatMapN(visitAnnotations(ann)) {
+        case Ast.Annotations(annotations) if annotations.nonEmpty =>
```

Comment:
I think we actually also check for duplicate annotations? Dont we?

Reply:
Yes, but that's handled by `visitAnnotations`

Reply:
I have a test for duplicate annotations